<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowOneSection</class>
 <widget class="QWidget" name="ShowOneSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>371</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChooseSection">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Название секции</string>
   </property>
   <widget class="QComboBox" name="comboBoxSection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
